src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1893,12 +1955,10 @@ protected override void HandleEvaluateRequestAsync(IRequestResponder<EvaluateArg
             hr = frame.GetExpressionContext(out expressionContext);
             eb.CheckHR(hr);
 
-            const uint InputRadix = 10;
-            DAPEvalFlags dapEvalFlags = DAPEvalFlags.NONE;
             IDebugExpression2 expressionObject;
             string error;
             uint errorIndex;
-            hr = expressionContext.ParseText(expression, enum_PARSEFLAGS.PARSE_EXPRESSION, InputRadix, out expressionObject, out error, out errorIndex);
+            hr = expressionContext.ParseText(expression, enum_PARSEFLAGS.PARSE_EXPRESSION, Constants.EvaluationRadix, out expressionObject, out error, out errorIndex);
```

Comment:
>EvaluationRadix [](start = 101, length = 15)

nit: I might suggest calling this `InputRadix` or `ParseRadix` since `EvaluationRadix` doesn't indicate what this does.

Reply:
Oh, I see. Don't reuse `EvaluationRadix` here. The input to `ParseText` should always be `10`. AD7 was a bit crazy and supported something that we will never use - having a different input radix. So, in other words, some debugger where evaluating `10 + 6` would result in 22 (0x16) rather than 16. Instead make a new constant for `InputRadix` since someday we might want to fix MIEngine to allow hex output (it is part of the DAP, but VS Code doesn't send it). But we will never allow hex input.

---
In reply to: [444418266](https://github.com/microsoft/MIEngine/pull/1013#discussion_r444418266) [](ancestors = 444418266)